### ISSUE TITLE
Evolution tasks view calendar popover: day highlight hard to read

### ISSUE THREAD

**Issue by nicorikken** [ubuntu/yaru#660]
### Readability issue

First off: thank you all for this awesome theme! It is very well polished.

When editing a task item in Evolution in a dedicated edit window, the date can be inserted using a popover. The selected day is highlighted with a white boxed line, which is hard to read:

![calendar-highlight-issue](https://user-images.githubusercontent.com/7458098/43321646-d9cf24b4-91ac-11e8-8d2d-2e406ee5473f.jpg)

In contrast, other calendar view use an orange square to highlight the selected date. I've check a couple:
- Task date selector from the main task view (listing).
- Appointment editor
- Meeting editor
- Contact editor (Anniversary & Birthday)

![calendar-highlight-better](https://user-images.githubusercontent.com/7458098/43321645-d990bbca-91ac-11e8-801c-d24ce0b760f6.jpg)

*The screenshot feature was disabled whilst having the popover opened up, so you'll have to cope with these photographs.*

**Note:** The misaligned arrows for month/year selection in this popup bug me as well, but I think that should be addressed in the Evolution project. When the popover is wide enough, it renders nicely, otherwise they cross over.

**Comment by clobrano** [comment]
I am afraid this is evolution using some theme colors for its own stuff (I should look into the code to know better), because it's the same on Adwaita

![evolution-calendar-popover](https://user-images.githubusercontent.com/2883614/43356376-088d13ec-9270-11e8-8c1e-eb63c505de04.gif)

just by chance, Adwaita's background is darker so the white selection "ring" (it's actually a square) is more visible, and it's selection color (blue) suits better in this use case.

However we usually prefer to not provide any per-app theme style.

**Comment by git-hardwareker** [comment]
Hello together, 
I would like to speak about this old and still valid topic again (open a new task?).
There are a lot of topics with the colours and bad readability
(like https://gitlab.gnome.org/GNOME/evolution/-/issues/1777).
clobrano wrote "we cannot change this behavior". Why not? Why not starting an style-sheet in a separate file instead of having the styles hard-coded? Ore are there other solutions to make it possible to change the colours for users ?

**Comment by clobrano** [comment]
> Why not? Why not starting an style-sheet in a separate file instead of having the styles hard-coded?

Yes sure, but this is a change on Evolution side, we cannot do it from Yaru
